config/routes/api.rb

```diff
@@ -27,6 +32,7 @@
 
         resource :history, only: :show
         resource :source, only: :show
+        resources :reactions, only: [:update, :destroy]
```

Comment:
Where does this come from?
This was in neither of the earlier PRs and I cannot find the commit for this change.

Reply:
Interesting, this was part of a commit modifying routes.rb in the original patches, but seems to have later been removed. I can't pinpoint this, but I ran over it during rebasing.

Reply:
Ah, it was added by 9fccc8ca7633f340f7809b9ff79226df327829bf and removed in be710b6a5f87df5d3b3392b597554f13374aec07